packages/grid/x-data-grid-pro/src/tests/editComponents.DataGridPro.old.test.tsx

```diff
@@ -852,14 +852,15 @@ describe('<DataGridPro /> - Edit Components', () => {
       fireEvent.click(input);
       fireEvent.doubleClick(getCell(1, 0));
 
+      expect(cell).to.have.class('MuiDataGrid-cell--editing');
       await new Promise((resolve) => nativeSetTimeout(resolve)); // Wait for promise
 
       expect(preProcessEditCellProps.callCount).to.equal(1);
       expect(preProcessEditCellProps.lastCall.args[0].props).to.deep.equal({
         value: true,
         isValidating: true,
       });
-      expect(cell).to.have.class('MuiDataGrid-cell--editing');
+      expect(cell).not.to.have.class('MuiDataGrid-cell--editing');
```

Comment:
The old assertion was wrong. After double-clicking another cell, the first cell should leave the edit mode. This is how it works outside a test environment. With the changes I did here this problem became apparent.